cmd/verify/network/cmd.go

```diff
@@ -153,8 +152,19 @@ func run(cmd *cobra.Command, _ []string) {
 			os.Exit(1)
 		}
 	} else if !cmd.Flags().Changed(statusOnlyFlag) {
-		r.Reporter.Errorf("%s is required", roleArnFlag)
-		os.Exit(1)
+		if cmd.Flags().Changed(clusterFlag) {
+			if cluster == nil {
+				cluster = r.FetchCluster()
+			}
+			if args.roleArn = cluster.AWS().STS().RoleARN(); args.roleArn == "" {
+				r.Reporter.Errorf("Unable to get role ARN from non-STS cluster '%s'.",
```

Comment:
I dont think this error is clear, should just return something like `NV is only supported for STS clusters`. 

Pretty much if roleARN is empty is the toggle indicating if a cluster is STS or not.

Reply:
updated, follow-on PRs will handle non-sts as well. thanks!